css/style.css

```diff
@@ -0,0 +1,735 @@
+}
+
+.project {
+    position: relative;
+    background-color: #003B4B;
+    border-radius: 10px;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+}
+
+.project h2 {
+    color: #FFFFFF;
+    font-family: "Copse", serif;
+    font-weight: 400;
+    font-size: 20px;
+    width: 100%;
+    padding-bottom: 20px;
+    text-align: center;
+    border-bottom: 1px solid #D5E3E7;
+}
+
+.project p {
+    color: #D5E3E7;
+    padding-top: 20px;
+    font-family: "Delius", cursive;
+    font-weight: 400;
+    font-size: 18px;
+}
+
+.project a img {
+    position: absolute;
+    top: 15px;
+    right: 15px;
+}
+
+.contact-me-page {
+    width: 100%;
+    height: auto;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 20px 0;
+}
+
+.contact-left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+}
+
+.contact-left-title h2 {
+    font-family: "Copse", serif;
+    font-weight: 600;
+    font-style: normal;
+    color: #588CA4;
+    font-size: 40px;
+    margin-bottom: 5px;
+}
+
+.contact-left-title hr {
+    border: none;
+    width: 120px;
+    height: 2px;
+    background-color: #588CA4;
+    border-radius: 10px;
+    margin-bottom: 20px;
+}
+
+.contact-inputs {
+    width: 400px;
+    height: 50px;
+    border: 1px solid #588CA4;
+    outline: none;
+    padding: 0 25px;
+    font-family: "Delius", cursive;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 16px;
+    color: #666;
+    border-radius: 50px;
+}
+
+.contact-left textarea {
+    height: 140px;
+    padding: 15px 25px;
+    border-radius: 20px;
+}
+
+.contact-inputs:focus {
+    border: 2px solid #588CA4;
+}
+
+.contact-inputs::placeholder {
+    color: #a9a9a9;
+}
+
+.contact-left button {
+    display: flex;
+    align-items: center;
+    padding: 15px 30px;
+    color: #fff;
+    gap: 10px;
+    border: none;
+    border-radius: 50px;
+    background: linear-gradient(270deg, #36454b, #588CA4);
+    cursor: pointer;
+}
+
+.contact-left button img {
+    height: 15px;
+    width: 15px;
+}
+
+.footer-section {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 15vh;
+    background-image: url("../images/footer/footer-background.gif");
+    background-size: cover;
+    background-repeat: no-repeat;
+    color: #FFFFFF;
+    padding: 0 40px;
+}
+
+.footer-section::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 59, 75, 0.799);
+    z-index: 1;
+}
+
+.footer-section div {
+    z-index: 2;
+}
+
+.footer-icons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+}
+
+.footer-icons img {
+    width: 30px;
+    height: 30px;
+}
+
+/* Media Queries */
```

Comment:
**Standard Breakpoints**
try to use commonly agreed-upon breakpoints for consistency. However, these can be adjusted depending on your design needs:

1. 320px – 480px: Mobile devices
2. 481px – 768px: Tablets (portrait)
3. 769px – 1024px: Tablets (landscape) and small desktops
4. 1025px – 1440px: Desktops

Reply:
Resolved.

I have refactored the code to adapt these standards as possible, but I still have some breakpoints based on content for some cases that might not even happen, but to make sure it's a responsive website as possible.